### ISSUE TITLE
Inconsistency in double_space reporting

### ISSUE THREAD

**Issue by FreHu** [abaplint/abaplint#1559]
After character literals double\_space is not reported, but after string literals it is.

```abap
    APPEND VALUE #( content = '201834'            format = 'YYYYMMDD'           exception = abap_true ) TO tested_set.
    APPEND VALUE #( content = |20180304 |         format = |YYYYMMDD | output = '20180304' ) TO tested_set.
```

![](https://user-images.githubusercontent.com/5097067/99878328-dcc01800-2c04-11eb-8484-777fe6837450.png)
